cypress/e2e/checkin/check-in-claimed-returned-items.cy.js

```diff
@@ -101,8 +101,6 @@ describe('Check In - Actions', () => {
       [
         permissions.checkinAll.gui,
         permissions.loansAll.gui,
```

Comment:
this permission: Users: User loans view, change due date, renew but you need a permission only to view, please choose the permission with the string that I specified above in the comments

Reply:
done